### PR TITLE
Add displyTask, clear, addItemUI test functions

### PR DESCRIPTION
**Pull request by farouqdaif1**
Tested the following functions:
- displyTask()
 - clear()
- addItemUI()

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/uImanpulate.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  },
+];
+
+describe('tests for UI ', () => {
+  describe('Test for Ui', () => {
+    test('Add item to list', () => {
+      addItemUI(task);
+      expect(list.childElementCount).toBe(1);
+    });
+  });
```
Comment (mikenath223) [inline src/test/uImanpulate.test.js:33]: - [ ] It seems you've failed to add tests for the following functionalities, kindly add them:
- Add test for editing a to-do item.
- Add test to update completed status of a to-do item.
- Add test for Clear all completed functionality.

Comment (Lockless404) [reply src/test/uImanpulate.test.js:33]: Edit function has been tested addRemove.test.js.

**Comment by farouqdaif1** [comment]
Edit function has been tested inside addRemove.test.js file.

**Review by helman101 — APPROVED** [review]
# STATUS: APPROVED 🟢 🥳 

Hi @farouqdaif1 🥇 , 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/uImanpulate.test.js (hunk 1 above)
Comment (farouqdaif1) [reply src/test/uImanpulate.test.js:33]: Thank you

**Pull request merged**